contracts/root/depositManager/DepositManager.sol

```diff
@@ -4,15 +4,18 @@ import { IERC721Receiver } from "openzeppelin-solidity/contracts/token/ERC721/IE
 import { IERC20 } from "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
 import { IERC721 } from "openzeppelin-solidity/contracts/token/ERC721/IERC721.sol";
 import { SafeMath } from "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import { Ownable } from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 
+import { BytesLib } from "../../common/lib/BytesLib.sol";
 import { ContractReceiver } from "../../common/misc/ContractReceiver.sol";
 import { Registry } from "../../common/Registry.sol";
 import { WETH } from "../../common/tokens/WETH.sol";
 import { IDepositManager } from "./IDepositManager.sol";
 import { DepositManagerStorage } from "./DepositManagerStorage.sol";
+import { StateSender } from "../state/StateSender.sol";
```

Comment:
The contract name is `StateSyncer` while file name is `StateSender.sol`. Let's rename the file to `StateSyncer.sol`